e2e/test/scenarios/models/reproductions.cy.spec.js

```diff
@@ -1174,6 +1175,202 @@ describe("issue 29517 - nested question based on native model with remapped valu
   });
 });
 
+describe("issue 53556 - nested question based on native model with remapped values", () => {
+  const questionDetails = {
+    name: "53556",
+    type: "model",
+    native: {
+      query:
+        "Select " +
+        'Orders."ID" AS "ID", ' +
+        'Orders."CREATED_AT" AS "CREATED_AT_ALIAS", ' +
+        'Orders."TOTAL" AS "TOTAL_ALIAS" ' +
+        "From Orders",
+      "template-tags": {},
+    },
+  };
+
+  beforeEach(() => {
```

Comment:
Wouldn't these tests be faster if this was (mostly) a `beforeAll()`? Maybe that doesn't work in Cypress-land, but in order to test a handful of drills against a model, the model only has to be created and configured once.

Reply:
Good point. For whatever reason (flake reduction over speed?), `beforeEach` seems much more common in the e2e tests. It looks like there are only ~10ish calls to `before()` in the whole suite, many in `describe` blocks with only a single test, vs. ~1,000ish calls to `beforeEach()`.

I tried switching this to `before` locally and it breaks tests 2..N because apparently the `@nestedQuestionId` is only wrapped and available in the first test. There's probably some cypress workaround for this. I guess I could just capture it in a regular `let` var scoped to the block or something, but given the overall prevalence of `beforeEach` I'm inclined to leave it.